sprints/dashboard/libs/google.py

```diff
@@ -166,3 +166,17 @@ def get_availability_spreadsheet() -> List[List[str]]:
             range=settings.GOOGLE_AVAILABILITY_RANGE,
             majorDimension='ROWS',
         ).execute()['values']
+
```

Comment:
If I'm not mistaken, we can simplify the whole future rotation composition to one line. Please see below. Therefore this whole function won't be needed.